stacks-common/src/types/chainstate.rs

```diff
@@ -34,18 +34,13 @@ pub type StacksPublicKey = Secp256k1PublicKey;
 pub type StacksPrivateKey = Secp256k1PrivateKey;
 
 /// Hash of a Trie node.  This is a SHA2-512/256.
+#[derive(Default)]
 pub struct TrieHash(pub [u8; 32]);
 impl_array_newtype!(TrieHash, u8, 32);
 impl_array_hexstring_fmt!(TrieHash);
 impl_byte_array_newtype!(TrieHash, u8, 32);
 impl_byte_array_serde!(TrieHash);
 
-impl Default for TrieHash {
```

Comment:
I'm curious why Clippy deleted this?

Reply:
It gets it for free using the above `#[derive(Default)]`. Clippy recognized that it could easily be derived and suggested using the keyword instead.